docs/installing-solidity.rst

```diff
@@ -9,11 +9,18 @@ Installing the Solidity Compiler
 Versioning
 ==========
 
-Solidity versions follow `semantic versioning <https://semver.org>`_ and in addition to
-releases, **nightly development builds** are also made available.  The nightly builds
-are not guaranteed to be working and despite best efforts they might contain undocumented
-and/or broken changes. We recommend using the latest release. Package installers below
-will use the latest release.
+Solidity versions mostly follow `Semantic Versioning <https://semver.org>`_.  We make
+an assurance in addition to SemVer that we will not introduce breaking changes into a patch
+level release. That means if you are using 0.x.y, then you can expect everything to work
+the same or better with 0.x.z with z > y.
```

Comment:
What about something like this?
```suggestion
Solidity versions follow `Semantic Versioning <https://semver.org>`_. We follow npm's
extension of Semantic Versioning in that before the 1.0.0 release, patch level releases
do not contain breaking changes. That means code that compiles with version 0.x.y
can be expected to compile with .x.z with z > y as well.
```

Reply:
Thank you, added. But also want to remove the npm note because [citation needed]

Reply:
Also updated with 39c8087d49876cfcfe5f10f02cda71e46795bd7e